include/dem/dem.h

```diff
@@ -278,6 +278,7 @@ class DEMSolver
   Particles::ParticleHandler<dim, dim> particle_handler;
   unsigned int                         contact_detection_step;
   unsigned int                         load_balance_step;
+  unsigned int                         checkpoint_step;
```

Comment:
Why are those variables not bool instead of ints?

Reply:
Changed to bool.